i18n/locales/de/common.json

```
@@ -1,238 +1,532 @@
+  "Your signed message": "Deine signierte Nachricht",
+  "Your voice matters": "Deine Stimme zählt.",
+  "You’ll see it in Delegates and it will be confirmed in a matter of minutes.": "You’ll see it in Delegates and it will be confirmed in a matter of minutes.",
+  "from": "von",
+  "or": "oder",
+  "{{deviceModel}} connected! Open the Lisk app on the device": "{{deviceModel}} verbunden! Öffne die Lisk App auf dem Gerät",
+  "{{fee}} {{token}}": "{{fee}} {{token}}",
+  "{{length}} bytes left": "{{length}} verbleibende Bytes",
+  "{{length}} extra characters": "{{length}} verbleibende Zeichen",
+  "{{length}} out of {{maxLength}} characters left": "{{length}} von {{maxLength}} verleibende Zeichen ",
+  "{{passphraseName}} paper wallet": "{{passphraseName}} Papier wallet",
+  "{{token}} Balance": "{{token}} Kontostand",
+  "{{token}} Wallet": "{{token}} Wallet",
+  "{{token}} balance": "{{token}} Kontostand",
+  "Account Balance:          {{balance}} {{token}}": "Kontostand:          {{balance}} {{token}}",
+  "Address already bookmarked": "Address already bookmarked",
+  "Become a delegate summary": "Become a delegate summary",
+  "Bookmarked": "Bookmarked",
+  "DD MMM YY, HH:mm": "DD MMM YY, HH:mm",
+  "DD MMM YYYY": "DD MMM YYYY",
+  "DD MMM YYYY, hh:mm A": "DD MMM YYYY, HH:mm",
+  "DD.MM.YY": "DD.MM.YY",
+  "Discord": "Discord",
+  "Discreet Mode Disabled": "Der Diskreter Modus ist deaktiviert",
+  "Discreet Mode Enabled": "Der Diskreter Modus ist aktiviert",
+  "Each LSK token is worth one vote.": "Each LSK token is worth one vote.",
+  "English": "Englisch",
+  "Filter by name...": "Filtern nach Namen...",
+  "German": "Deutsch",
+  "Go to Dashboard": "Gehe zum Dashboard",
+  "Go to Wallet": "Gehe zum Wallet",
+  "Go to confirmation": "Gehe zum Bestätigung",
+  "Go to settings": "Gehe zum Einstellungen",
+  "Invalid address": "Ungültige Adresse",
+  "Invalid dates": "Ungültiges Datum",
+  "MMM DD YYYY": "MMM DD YYYY",
+  "MMM DD YYYY hh:mm": "MMM DD YYYY hh:mm",
+  "MMM DD YYYY hh:mm:ss": "MMM DD YYYY hh:mm:ss",
+  "MMM DD YYYY hh[h]": "MMM DD YYYY hh[h]",
+  "MMM YYYY": "MMM YYYY",
+  "Passphrase:": "Passphrase",
+  "Removed votes": "Entfernte Stimmen",
+  "YYYY": "YYYY",
+  "Yesterday": "Gestern",
+  "You don’t have enough balance to enable it. {{fee}} LSK is required.": "You don’t have enough balance to enable it. {{fee}} LSK is required.",
+  "Your {{walletModel}} has an old Firmware. Please update it.": "Your {{walletModel}} has an old Firmware. Please update it.",
+  "[Today], hh:mm A": "[Heute], HH:mm",
+  "[Tomorrow], hh:mm A": "[Morgen], HH:mm",
+  "[Yesterday], hh:mm A": "[Gestren], HH:mm",
+  "ie. 192.168.0.1": "ie. 192.168.0.1",
+  "ie. Lisker123": "ie. Lisker123",
+  "ie. peter_pan": "ie. peter_pan",
+  "Just now": "gerade jetzt",
```

https://github.com/LiskHQ/lisk-hub/pull/2406#discussion_r321628685 still happens in German 
```suggestion
  "Just now": "Gerade jetzt",
```